### PR TITLE
[DOC]: NymVPN testing Guides update

### PR DESCRIPTION
**Pull request by serinko**
# Description
Routine sync of the nymvpn docs with the client development

### CODE REVIEW

File: documentation/dev-portal/src/nymvpn/cli-linux.md [hunk 1]
```diff
@@ -56,16 +51,16 @@ In case of errors, see [troubleshooting section](troubleshooting.md).
 
 The basic syntax of `nym-vpn-cli` is:
 ```sh
-sudo ./nym-vpn-cli -c ./sandbox.env --entry-gateway-id <ENTRY_GATEWAY_ID> --exit-router-address <EXIT_ROUTER_ADDRESS> --enable-wireguard --private-key <PRIVATE_KEY> --wg-ip <WG_IP>
+sudo ./nym-vpn-cli <--exit-router-address <EXIT_ROUTER_ADDRESS>|--exit-gateway-id <EXIT_GATEWAY_ID>|--exit-gateway-country <EXIT_GATEWAY_COUNTRY>>
```
Comment (mfahampshire) [inline documentation/dev-portal/src/nymvpn/cli-linux.md:54]: what's going on with the first `<--` here? I'm a little confused by it

Comment (mfahampshire) [reply documentation/dev-portal/src/nymvpn/cli-linux.md:54]: are these optional values?

File: documentation/dev-portal/src/nymvpn/cli-linux.md [hunk 2]
```diff
@@ -56,16 +51,16 @@ In case of errors, see [troubleshooting section](troubleshooting.md).
...
-* To choose different Gateways, visit [nymvpn.com/en/alpha/api/gateways](https://nymvpn.com/en/alpha/api/gateways)
+* To choose different Gateways, visit [nymvpn.com/en/alpha/api/gateways](https://explorer.nymtech.net/network-components/gateways)
 * To see all possibilities run with `--help` flag:
 ```sh
 ./nym-vpn-cli --help
 ```
 ~~~admonish example collapsible=true title="Console output"
 ```sh
-Usage: nym-vpn-cli [OPTIONS]
+Usage: nym-vpn-cli [OPTIONS] <--exit-router-address <EXIT_ROUTER_ADDRESS>|--exit-gateway-id <EXIT_GATEWAY_ID>|--exit-gateway-country <EXIT_GATEWAY_COUNTRY>>
```
Comment (mfahampshire) [inline documentation/dev-portal/src/nymvpn/cli-linux.md:63]: see prev comment

File: documentation/dev-portal/src/nymvpn/cli-linux.md (hunk 1 above)
Comment (serinko) [reply documentation/dev-portal/src/nymvpn/cli-linux.md:54]: Those are three option for one conditional arg, changed it.

**Review by mfahampshire — APPROVED** [review]

**Pull request merged**